docs/dashboard/orders.md

```diff
@@ -136,9 +138,21 @@ When order confirmation is enabled, order needs to be confirmed by staff in orde
 
 1. Select an unconfirmed order from order list to open the order details
 
-2. Click _confirm&nbsp;order_ button on the bottom
+2. There is a possiblity to edit the order at this point. However there are a few things to be aware when doing so:
+
+- When order address gets changed, existing allocations remains untouched.
+- New allocations are created whenever staff increase order line amount or add a new order line.
+- Allocations will only get altered for variants with active inventory tracking set.
```

Comment:
Here we could also link to the page when users can control tracking inventory for a variant.